draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -102,19 +102,19 @@ specification. {{sctp-Crypto-chunk-layering}} illustrates the CRYPTO
 chunk layering in regard to SCTP and the Upper Layer Protocol (ULP).
 
 ~~~~~~~~~~~ aasvg
++---------------+ +----------------------------------+
+|               | |           Protection Engine      |
+|      ULP      | +----------------+                 |
+|               | | Key Management |                 |
++---------------+-+---+----+-------+                 |
+|                     |    |                         |
+| SCTP Chunks Handler | +-- SCTP Unprotected Payload |
+|                     |/   |                         |
++---------------------+    +--------------------+    |
+|        CRYPTO       |    |  DATA Management   |    |
```

Comment:
I think the Data Management part should likely be called "Protection Opertions" or something similar. I am also uncertain about the box. I would think that having the protection engine on the top. And then have the box Protection operations that exchange protected data and clear text with the crypto block, and then have keys being pushed down from the upper layers. One realization the protection operations happens embedded in the crypto block.